### PR TITLE
adding get and set lr for optimizer

### PR DESCRIPTION
**Pull request by AdamLouly**
### Description
Exposing get and set Learning rate for optimizer

### Motivation and Context
you can now set learning rate for optimizer.

### CODE REVIEW

File: orttraining/orttraining/python/training/api/optimizer.py [hunk 1]
```diff
@@ -22,3 +22,15 @@ def step(self):
         Run Optimizer Step.
         """
         self._optimizer.optimizer_step()
+
+    def set_learning_rate(self, learning_rate):
```
Comment (baijumeswani) [inline orttraining/orttraining/python/training/api/optimizer.py:26]: let's add type hints if possible.

File: orttraining/orttraining/test/python/orttraining_test_python_bindings.py [hunk 2]
```diff
@@ -142,6 +142,29 @@ def test_optimizer_step():
...
+            temp_dir, simple_model, onnx_model, optimizer_model=optimizer_model
+        )
+        # Create Checkpoint State.
+        state = CheckpointState(checkpoint_file_path)
+        # Create a Module and Optimizer.
+        model = Module(model_file_path, state)
+        optimizer = Optimizer(optimizer_file_path, model)
+
+        # Test get and set learning rate.
+        lr = optimizer.get_learning_rate()
```
Comment (baijumeswani) [inline orttraining/orttraining/test/python/orttraining_test_python_bindings.py:161]: Can you test the default learning rate?

**Review by baijumeswani — DISMISSED** [review]

**Review by baijumeswani — APPROVED** [review]

**Comment by AdamLouly** [comment]
Thanks for the review
